docs/modules/ROOT/pages/delete-kafka-connection.adoc

```diff
@@ -0,0 +1,13 @@
+= Deleting a Data Connection to Kafka
+:description: You can delete a data connection if you no longer need it, or if you need to recreate the data connection with different details. This functionality is not currently available in the connector wizard but you can use SQL instead.
+:cloud-tags: Manage Data
+:cloud-title: Deleting a Data Connection to Kafka
+:cloud-order: 65
+
+{description}
```

Comment:
I think it would be good to add a warning here similar to DB case. 
````
Before you start, make sure that the data connection is no longer in use.
````

Reply:
Added as part of the include. I've checked and both sections use the same text.